docs/user-guides/capacity-plannig.md

```diff
@@ -0,0 +1,152 @@
+
+Each vald component has the default priority value:
+
+- Agent: 1000000000
+- Discoverer: 1000000
+- Filter Gateway: 1000000
+- LB Gateway: 1000000
+- Index Manager: 1000000
+
+Therefore, the order of priority is as follows:
+
+```bash
+Agent > Discoverer = Filter Gateway = LB Gateway = Index Manger
+```
+
+Those values will be helpful when the Pods other than the Vald component are in the same Node.
+
+It is easy to change by editing your Vald helm chart yaml.
+
+```yaml
+# e.g. LB Gateway podPriority settings.
+...
+  gateway:
+    lb:
+    ...
+      podPriority:
+        enabled: true
+        value: {new values}
+      ...
+```
+
+**QoS**
+
+QoS value can be either Guaranteed, Burstable, or BestEffort.
+And, QoS priority is higher in the order of Guaranteed, Burstable, BestEffort, and Kubernetes will kill Pods in ascending order of importance.
+
+Resource request and limit determine QoS.
+Vald requires many RAM resources because of on-memory indexing, so we highly recommend that you do not specify a limit, especially for the Vald Agent.
+In this case, QoS will be Burstable.
+
+In addition, when other components coexist with Vald Agent, it is preferable to set resource requests and limits considering QoS. 
+
+If it needs to set the resource limit, it would be better to set `podAntiAffinity` like below.
+
+```yaml
+# e.g. Agent podAntiAffinity settings
+...
+  agent:
+  ...
+    affinity:
+      nodeAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution: []
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms: []
+      podAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution: []
+        requiredDuringSchedulingIgnoredDuringExecution: []
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              topologyKey: kubernetes.io/hostname
+              labelSelector:
+              matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                    - vald-agent-ngt
+        requiredDuringSchedulingIgnoredDuringExecution: []
+  ...
+```
+
+**Throttling**
+
+The CPU throttling affects the pod performance.
+
+If it occurs, the Vald cluster operator must consider each component's CPU resource request and limit.
+It is easy to change by editing your values yaml file and applying it. 
+
+```yaml
+# e.g. LB Gateway resources settings.
+...
+  gateway:
+    lb:
+    ...
+      resources:
+        requests:
+          cpu: 200m
+          memory: 150Mi
+        limits:
+          cpu: 2000m
+          memory: 700Mi
+    ...
+```
+
+<div class="warning">
+Please take care of pod priority and QoS.
+</div>
+
+### Component view
+
+Depending on the customization of each component for each user, there are some points to be aware of.
+
+**Index Manager**
+
+If saveIndex is frequent, backup data per unit time will increase, which consumes bandwidth.
```

Comment:
```suggestion
If saveIndex is frequently executed, the backup data per unit time will increase, which consumes bandwidth.
```

Reply:
```suggestion
If the `saveIndex` executes frequently, backup data per unit time will increase, which consumes bandwidth.
```

Reply:
I think using passive sentence here is better 🤔

Ref:
```
when explaining something in which the reader is not actively involved in or when explaining a process, Passive Voice can be the more appropriate option.
```
https://medium.com/@DaphneWatson/technical-writing-active-vs-passive-voice-485dfaa4e498#:~:text=When%20Passive%20Voice%20is%20used,included%20in%20the%20web%20application

Reply:
```suggestion
If the `saveIndex` is executed frequently, the backup data per unit time will increase, which consumes bandwidth.
```